docresources/SyncBot/syncbot-quickstart.md

```diff
@@ -8,12 +8,16 @@ A little Bot, using the ProcBot framework, that links communication services.
 * Notifications where you want them
 	* To solve for the problem of not noticing Front notifications
 	* But please, receive notifications somewhere
-* Brings Front "pro features" to the forums
+* Unlock Front "pro features" on Forum traffic
 	* Monitoring
 	* GitHub linked issues
 	* Audits
 	* Analytics
+* Control Front workflow from anywhere
+	* `#close` in a whisper will close the Front ticket
```

Comment:
`#close` or `#archive` right? Also in line 60 I think there's a `#reply` mention